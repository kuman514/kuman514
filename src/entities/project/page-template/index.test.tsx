import { render, screen } from '@testing-library/react';

import ProjectPageTemplate from '.';

describe('ProjectPageTemplate', () => {
  it('should contain most recently modified date', async () => {
    render(
      <ProjectPageTemplate mostRecentlyEdited={new Date('2024-08-01')}>
        Lorem ipsum dolor sit amet.
      </ProjectPageTemplate>
    );

    expect(
      await screen.findByText(/최근 수정 일자: 2024년 8월 1일/i)
    ).not.toBeNull();
  });

  it('should match the snapshot', () => {
    const { container } = render(
      <ProjectPageTemplate mostRecentlyEdited={new Date('2024-08-13')}>
        Lorem ipsum dolor sit amet.
      </ProjectPageTemplate>
    );

    expect(container.firstChild).toMatchSnapshot();
  });
});
