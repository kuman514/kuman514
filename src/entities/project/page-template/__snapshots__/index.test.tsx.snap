// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProjectPageTemplate should match the snapshot 1`] = `
<main
  class="min-h-screen flex flex-col w-full flex flex-col items-center"
>
  <div
    class="flex flex-col px-8 py-24 gap-16 sm:px-24 max-w-6xl"
  >
    <div
      class="w-full text-right"
    >
      최근 수정 일자: 
      2025
      년 
      8
      월
       
      9
      일
    </div>
    Lorem ipsum dolor sit amet.
  </div>
</main>
`;
